### DISCUSSION TITLE
In the new version, the layout of materials in the blog is broken

### DISCUSSION THREAD

**Discussion by designer-dk** [templaza/astroid-framework#857] [General]
Hello.
I specify a layout of 3 material cards in a row for displaying materials on my blog. In the new version of Astroid, all materials are shown as a very long column one material card wide.
If I change the layout to 4 in a row or any other layout, it doesn't change anything.

**Comment by felixkat** [comment]
Can you provide a screenshot?

Is it articles you are displaying?  Make sure you add an entry for columns

![image](https://github.com/user-attachments/assets/15642da8-a6f8-45b6-995a-55fa6a009dc5)

**Reply by designer-dk** [reply]
I'd really hate to do that, as I'd have to roll back the site from backup afterward. But I understand and will do it tomorrow.
Now I am using Astroid Framework 3.0.15 and One Template 3.0.12 and everything goes fine.

![3-in-a-row](https://github.com/user-attachments/assets/0abe72eb-8692-4869-8006-1a3ed4dff933)

**Comment by felixkat** [comment]
By default the menu option of Articles-> Category Blog->Blog Layout is set to, "1 column", this is the default.  It's possible that you had it set to 3 before and the update has reset it. 

![image](https://github.com/user-attachments/assets/9312d5df-258e-458f-9153-8252c06f8549)

**Reply by designer-dk** [reply]
Take my word for it: I've customized the blog many times and have an idea of how it's done :)

Below in two screenshots is the story of the upgrade to the new version of Astroid. As you can see, everything turns into a mess very quickly.
But I've already rolled back and everything is fine again :)

![old_astroid](https://github.com/user-attachments/assets/bbe154dc-5a42-4a2d-8580-81a6e72cbc4a)
![new_astroid](https://github.com/user-attachments/assets/fb55d1fa-4b3f-431c-b1ec-5a248c4b4f96)

**Reply by sonvnn** [reply]
Please help me click Astroid Clear Cache in the administrator to recompile style. If this issue is still happen please send me your url via sonlv@templaza.com. I will help you check it.

Thanks & best regards,
Sonny

**Reply by designer-dk** [reply]
Thanks for your support. I have updated Astroid again and reset the cache several times: framework, CMS and browser. Used two other browsers as well :) The problem remained.
Also, the pages started to slow down when scrolling, and the difficulty with Firefox and the new library has already been discussed here.

**Comment by felixkat** [comment]
Unfortunately I can't recreate this issue.  I installed Joomla 4 and Astroid 3.0.15 with the one template.  I created 6 articles and setup a menu link matching your screenshot.

I then updated to Astroid 3.1.7 and one template 3.1.0 and everything is still working okay.

![image](https://github.com/user-attachments/assets/1c944bdf-78bf-48d7-87b3-57c3a1a619a1)


It's probably best that Sonny has a look at your website.

**Reply by sonvnn** [reply]
I think most of the time this issue is due to user editing some things in template or child template. And when things are upgraded they are not compatible with the latest version.

**Reply by designer-dk** [reply]
Totally agree with you. But not in this case: I didn't do any overrides for the blog layout. In other places - yes, maybe it had some effect. But it's strange, because previous Astroid updates installed without problems. I'll check the overrides again, of course.

![Untitled-1](https://github.com/user-attachments/assets/72701f70-4a0a-4755-8f0d-3580f994d3b6)

**Reply by designer-dk** [reply]
> I installed Joomla 4 and Astroid 3.0.15 with the one template. I created 6 articles and setup a menu link matching your screenshot.
> I then updated to Astroid 3.1.7 and one template 3.1.0 and everything is still working okay.

I have Joomla 5.2 installed. I wonder if it could be that?

**Reply by felixkat** [reply]
To be honest I used Joomla 4 as it was an older version of Astroid and I just assumed at the time the initial installation would have been Joomla 4.

I will upgrade to Joomla 5 and test again.

**Comment by designer-dk** [comment]
Hello again.
Coming back to this story again, sorry :)

![Untitled-1](https://github.com/user-attachments/assets/e32f1ac9-3c22-4d3e-8fdf-2a7f631c162f)

Tested the idea: I made a backup of the blog layout overrides used by Astroid Framework 3.0.15. Upgraded to 3.1.7 and overwrote the new overrides with the ones I saved in the backup. The blog layout started to look great again, as it should.
I still have a problem with the mouse wheel scrolling not working in Firefox after the upgrade, but that's another issue.
